src/control/server/config_types.go

```diff
@@ -227,6 +227,7 @@ type configuration struct {
 	NrHugepages    int             `yaml:"nr_hugepages"`
 	ControlLogMask ControlLogLevel `yaml:"control_log_mask"`
 	ControlLogFile string          `yaml:"control_log_file"`
+	Username       string          `yaml:"username"`
```

Comment:
Admins might want to run the program with a separate gid as well. I would put User/Group as the config options.

Reply:
sure